hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHttpServer.java

```diff
@@ -64,16 +64,14 @@ public static Collection<Object[]> policy() {
   }
 
   @BeforeAll public static void setUp() throws Exception {
-    File base = new File(BASEDIR);
```

Comment:
Here we can minimize change by adding `@TempDir File base` as parameter, instead of adding a member variable.

Reply:
Done. thanks.